MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentComponentsController.swift

```diff
@@ -802,6 +804,8 @@ extension PaymentComponentsController: PaymentReviewProtocol {
     }
     
     public func presentShareInvoiceBottomSheet(paymentRequestId: String, paymentInfo: GiniInternalPaymentSDK.PaymentInfo) {}
+    
+    public func paymentReviewClosed() {}
```

Comment:
[clarification] Normally we should have a comment inside like " // Default implementation" or something like this. Can this be a case like this? Sonarcloud will complain about this.

Reply:
On MerchantSDK it's not yet implemented/updated with the latest flow. This is TODO